tests/test_external_api.py

```diff
@@ -0,0 +1,21 @@
+from src.external_api import currency_conversion, API_KEY
+from unittest.mock import patch
+
+
+def test_currency_conversion(transaction_api):
+    """Тест на проверку входных данных RUB"""
+    rub_transaction = currency_conversion(transaction_api)
+    assert rub_transaction == "31957.58"
```

Comment:
Должно быть так:
```suggestion
    assert rub_transaction == 31957.58
```

Проблема в том, что ты тест подгоняешь под функцию, а нужно наоборот. В данном случае по тесту видно, что функция работает не верно и должна возвращать float значение, но этого не происходит. Очевидно, что нужно править функцию